src/Azure.Functions.Cli/Diagnostics/ConsoleLoggerFactoryBuilder.cs

```diff
@@ -22,7 +22,7 @@ public void AddLoggerProviders(ILoggerFactory factory, ScriptHostConfiguration s
             }
 
             // TODO: Get trace level, add console logger
-            // scriptConfig.HostConfig.LoggerFactory.AddColoredConsole((n, l) => l >= LogLevel.Information, false);
+            scriptConfig.HostConfig.LoggerFactory.AddColoredConsole((cat, lvl) => cat.StartsWith("Worker."), false);
```

Comment:
shouldn't it be an `||` of the 2 conditions? or is `.startsWith("Worker.")` includes the other one as well?

Reply:
That sounds reasonable. I was whitelisting only worker stuff, but more logs of all kinds should be helpful at this stage

Reply:
oh I just realized that the whole line was commented. I thought you just changed the condition. I merged it as is and we can add that latter. I'll make a release shortly